### PR TITLE
Screenshot tests: Improve Activities activity tests

### PR DESCRIPTION
**Pull request by AlvaroBrey**
Force-close snackbar to not have the timing break tests

- [x] Tests written, or not not needed

### CODE REVIEW

File: src/androidTest/java/com/nextcloud/client/ActivitiesActivityIT.kt [hunk 1]
```diff
@@ -51,9 +50,9 @@ class ActivitiesActivityIT : AbstractIT() {
         shortSleep()
         Espresso.onView(withId(R.id.drawer_layout)).perform(DrawerActions.open())
         sut.runOnUiThread {
-            sut.binding.emptyList.emptyListView.visibility = View.VISIBLE
-            sut.binding.list.visibility = View.INVISIBLE
+            sut.dismissSnackbar()
         }
+        shortSleep()
```
Comment (tobiasKaminsky) [inline src/androidTest/java/com/nextcloud/client/ActivitiesActivityIT.kt:55]: shortSleep might cause flaky tests, if emulator is too slow.
#internet recommends therefore waitForIdleSync().
Can you test if this also works?

Comment (AlvaroBrey) [reply src/androidTest/java/com/nextcloud/client/ActivitiesActivityIT.kt:55]: Added `waitForIdleSync()` after the `shortSleep` calls, but if I try to remove the `shortSleep` I get this error randomly:

```
        java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
        at com.facebook.testing.screenshot.layouthierarchy.LayoutHierarchyDumper.dumpHierarchy(LayoutHierarchyDumper.java:114)

```

Comment (tobiasKaminsky) [reply src/androidTest/java/com/nextcloud/client/ActivitiesActivityIT.kt:55]: Strange, so for some reasons it does not wait long enough, maybe as it is on a different thread?

Comment (AlvaroBrey) [reply src/androidTest/java/com/nextcloud/client/ActivitiesActivityIT.kt:55]: No idea :\

Comment (tobiasKaminsky) [reply src/androidTest/java/com/nextcloud/client/ActivitiesActivityIT.kt:55]: Ok. Then let us keep it this way.

**Review by tobiasKaminsky — APPROVED** [review]

**Pull request merged**